index.html

```diff
@@ -13,13 +13,20 @@
   </head>
   <body>
     <header class="header">
-      <a href="#" class="toolbar logo"><img src="images/logo.png" alt="" /></a>
-      <nav class="toolbar menu"><img src="images/menu.png" alt="" /></nav>
+      <div class="mobile-nav">
+        <a href="#" class="toolbar logo"><img src="images/logo.png" alt="" /></a>
+        <label>&#9776;</label>
+      </div>
+      <div class="desk-nav">
+        <a href="#">Portfolio</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </div>
     </header>
-    <section class="headline">
-      <div class="bg-icons">
-        <div class="bg"></div>
-        <div class="top">
+    <main class="main-sec">
+     <section class="top">
+      <div class="bg">
+       <div class="align-top">
           <div class="description">
             <h1>I'm Victor<br>Glad to see you!</h1>
             <p class="my-description">
```

Comment:
- [ ] Kindly be noticed that you should make your design like the Figma design.

Your Design | Figma Design
---------------- | -------------------
![Screenshot from 2022-06-03 01-37-28](https://user-images.githubusercontent.com/33752735/171758020-2a72471e-408d-400b-8cf9-caaf1a70f501.png) | ![Screenshot from 2022-06-03 01-37-59](https://user-images.githubusercontent.com/33752735/171758037-30d736c9-320a-4166-8c38-93f83cb471c9.png)

Your Design | Figma Design
---------------- | -------------------
![Screenshot from 2022-06-03 01-41-24](https://user-images.githubusercontent.com/33752735/171758174-12e53418-7115-4e65-b71d-d384bb8369e3.png) | ![Screenshot from 2022-06-03 01-41-07](https://user-images.githubusercontent.com/33752735/171758193-e3231772-eff6-4099-b164-3b22f9b6a1c1.png)